src/index.js

```diff
@@ -1,11 +1,10 @@
 const Stripe = require("stripe");
 
-const stripe = Stripe(STRIPE_API_KEY, {
-  // Cloudflare Workers use the Fetch API for their API requests.
-  httpClient: Stripe.createFetchHttpClient()
-});
-
-async function handleRequest(request) {
+async function handleRequest(request, env) {
+  const stripe = Stripe(env.STRIPE_API_KEY, {
```

Comment:
Can we move this back out of the async function?

Reply:
Unfortunately not, no.

Cloudflare Workers modules format doesn't support secrets as globals (note the change to `env.STRIPE_API_KEY`). They're where everything is moving towards because they're faster and safer to run. More info: https://developers.cloudflare.com/workers/learning/migrating-to-module-workers/